__tests__/tripRequestTest/tripRequestView.test.js

```diff
@@ -0,0 +1,213 @@
+import React from 'react';
+import { shallow, mount } from 'enzyme';
+import { TripRequest, mapStateToProps } from '../../src/views/triprequest.view';
+import sinon from 'sinon';
+import { applyMiddleware, createStore } from 'redux';
+import thunk from 'redux-thunk';
+import { props } from '../../__mockData__/tripRequest-mock-data';
+import reducer from '../../src/reducers/index';
+
+import { Provider } from 'react-redux';
+
+const middlewares = [thunk];
+const testStore = state => {
+	const createStoreWithMiddleware = applyMiddleware(...middlewares)(
+		createStore,
+	);
+	return createStoreWithMiddleware(reducer, state);
+};
+const initialSetUp = (initialState = {}) => {
+	const store = testStore(initialState);
+	const wrapper = mount(
+		<Provider store={store}>
+			<TripRequest {...props} />
+		</Provider>,
+	);
+	return wrapper;
+};
+const setUp = (initialState = {}) => {
+	const store = testStore(initialState);
+	const wrapper = shallow(<TripRequest {...props} store={store} />);
+	return wrapper;
+};
+
+describe('Trip Request component tests', () => {
+	it('should render Trip Request view page successfully', () => {
```

Comment:
@Jajabenit250 Probably this test case is not important. Kindly check what happens when you remove it and confirm.